PaintHouse.java

```diff
@@ -0,0 +1,31 @@
+// Time Complexity : O(N) or O(n*3) where n are houses and 3 is colors
```

Comment:
It will be O(N^2)

Reply:
Hi Jeel,
In the second loop, n is constant i.e. 3. So shouldn't it be O((m-1)*3) or O(N)?

Reply:
Although nested loop is there right!!!

Reply:
After expanding the term it is O(N) .

Reply:
Understood. Thanks for the clarification Jeel.